### ISSUE TITLE
remove the dependence on rospy

### ISSUE THREAD

**Issue by Peter9606** [bponsler/ros_comm#5]
Mainly below two reasons make message_filters still have dependence on rospy.

1. Porting of python log API is not ready
2. Porting of rospy.Timer is not ready

Fortunately, they will be ready in [Beta 3](https://github.com/ros2/ros2/wiki/Roadmap). So I'm going to do the modification respectively after that.

**Comment by Peter9606** [comment]
@bponsler what do you think about the "rospy"? Is the above okay with you?

**Comment by bponsler** [comment]
Yes, that makes sense to me. The logging capabilities would be easy to temporarily port to basic prints, but probably not worth it unless it is holding up further development.

**Comment by Peter9606** [comment]
Let's keep it open and handle it when beta 3 is released.
